mix.exs

```diff
@@ -96,14 +97,15 @@ defmodule DotCom.Mixfile do
       {:floki, "0.37.0"},
       {:gen_stage, "1.2.1"},
       {:gettext, "0.26.2"},
-      {:hackney, "1.20.1"},
-      {:hammer, "6.2.1"},
+      {:hackney, "1.23.0"},
+      {:hammer, "7.0.1"},
       {:html_sanitize_ex, "1.4.3"},
-      {:httpoison, "2.2.1"},
+      {:httpoison, "2.2.2"},
       {:inflex, "2.1.0"},
       {:jason, "1.4.4", override: true},
       {:kino_live_component, "0.0.5"},
       {:logster, "1.1.1"},
+      # reverted from 0.4
```

Comment:
I'd add a comment as to why but I don't remember and I don't want to revisit the trouble I had with that upgrade when I tried it a number of months ago (and forgot to document it then)